typings/css-properties.d.ts

```diff
@@ -5,6 +5,10 @@ type CSSWideKeyword = 'initial' | 'inherit' | 'unset'
 
 // Based on [CSSProperties from React types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts)
 export interface CSSProperties {
+   /**
+   * Glamorous-specific property which contains any of the following CSS definitions
+   */
+  css?: CSSProperties
```

Comment:
Since this prop is glamorous specific. I don't think it should be in here. Instead, you can have an intersection type [here](https://github.com/paypal/glamorous/blob/master/typings/glamorous.d.ts#L32)